### DISCUSSION TITLE
Need help with translating `README.md` into different languages

### DISCUSSION THREAD

**Discussion by gomzyakov** [gomzyakov/github-achievements#79] [General]
First of all these:

- [ ] Bengali
- [ ] Japanese
- [ ] Yue Chinese
- [ ] Vietnamese
- [ ] Turkish
- [ ] Wu Chinese
- [ ] Marathi
- [ ] Telugu
- [ ] Western Punjabi
- [ ] Korean
- [ ] Tamil
- [ ] Egyptian Arabic
- [ ] Urdu
- [ ] Javanese
- [ ] Iranian Persian
- [ ] Gujarati
- [ ] Hausa
- [ ] Bhojpuri
- [ ] Levantine Arabic
- [ ] Mongolian

**Comment by OxYGC** [comment]
Can it be translated into Mongolian?

**Reply by gomzyakov** [reply]
Yes, of course. If you have the opportunity, you can make a PR with translation.

### UPD

If there is no possibility, I added Mongolian to the list for translation.

**Comment by 0x000Akbar** [comment]
Can it be translated into Indonesian?

**Reply by gomzyakov** [reply]
@0x000Akbar Yes, of course! You can make a translation and open a PR)

**Reply by 0x000Akbar** [reply]
Ok, thank you bro 🙏
